packages/user-registry/src/blockchain-facade/Users/User.ts

```diff
@@ -1,37 +1,28 @@
 import { Configuration, BlockchainDataModelEntity } from '@energyweb/utils-general';
+import { IUserWithRelationsIds } from '@energyweb/origin-backend-core';
+
 import { Role } from '../../wrappedContracts/RoleManagement';
 import UserOffChainPropertiesSchema from '../../../schemas/UserOffChainProperties.schema.json';
 
-export interface IUserOffChainProperties {
-    email: string;
-    firstName?: string;
-    surname?: string;
-    street?: string;
-    number?: string;
-    zip?: string;
-    city?: string;
-    country?: string;
-    state?: string;
-}
+export type IUserOffChainProperties = {};
```

Comment:
Can this be removed then?

Reply:
I have experienced problems with trying to remove user offchain properties and `MarketUser` actually uses offchain properties (thus, for example, a need to leave logic in Solidity UserLogic).

I'd rather have this dummy type, as user-registry is going to change a lot soon and I see no harm in leaving 1 interface that is empty object - especially that is still referenced in multiple places where `User.Entity` is created - and that is problematic to due to being extended from `MarketUser.Entity`.

One of the problems I've experienced was with `prepareEntityCreation` where user off chain properties were empty.